api/handlers/container/inspect_test.go

```diff
@@ -56,15 +56,15 @@ var _ = Describe("Container Inspect API", func() {
 	})
 	Context("handler", func() {
 		It("should return inspect object and 200 status code upon success", func() {
-			service.EXPECT().Inspect(gomock.Any(), cid).Return(&resp, nil)
+			service.EXPECT().Inspect(gomock.Any(), cid, false).Return(&resp, nil)
```

Comment:
nit: also add a test here when size is `true`

Reply:
updated with testcase